shared/containers/Core/Core.js

```diff
@@ -29,9 +29,15 @@ export default class Core extends Component {
       .off('order update', this.updateOrders)
       .off('remove order', this.updateOrders)
       .off('new order request', this.updateOrders)
-    SwapApp.services.room.connection
-      .off('peer joined', actions.ipfs.userJoined)
-      .off('peer left', actions.ipfs.userLeft)
+    if (SwapApp.services.room.connection) {
+      console.log('leave room');
+      SwapApp.services.room.connection
```

Comment:
```suggestion
      SwapApp.services.room
        .off('peer joined', actions.ipfs.userJoined)
```

по-моему, этот код должен работать ^

Если нет разницы, то лучше через off, мне кажется

Reply:
там нет функции off

Reply:
у room есть off а у connection нет)